pdns/syncres.hh

```diff
@@ -1054,6 +1054,10 @@ public:
   string reason; //! Print this to tell the user what went wrong
 };
 
+class PolicyHitException
```

Comment:
Maybe extend from `PDNSException`?

Reply:
I'm not sure it's a good idea, the new exception might get caught in unexpected places, which should not happen with a new, unrelated exception.

Reply:
Right